Update breakpoints.mdx with warning about minifcation with remote debugging (#19849)
cloudflare/cloudflare-docs: src/content/docs/workers/observability/dev-tools/breakpoints.mdx

Breakpoint debugging in `wrangler dev` using `--remote` could extend Worker CPU time and incur additional costs since you are testing against actual resources that count against usage limits. It is recommended to use `wrangler dev` without the `--remote` option. This ensures you are developing locally.

> Additionally note that if you are debugging using `--remote` you cannot use code minification as the debugger will be unable to find vars when stopped at a breakpoint. Do not set minify to `true` in wrangler.toml/.json.

Reviewer: ```suggestion
If you are debugging using `--remote`, you cannot use code minification as the debugger will be unable to find vars when stopped at a breakpoint. Do not set minify to `true` in your Wrangler configuration file.
```
